runtime/binding-http/src/main/java/io/aklivity/zilla/runtime/binding/http/internal/config/HttpRequestType.java

```diff
@@ -29,6 +30,11 @@ public class HttpRequestType
     public HttpRequestConfig.Method method;
     public List<String> contentType;
 
+    // path parsing helpers
+    public Pattern pathPattern;
+    public Map<String, String8FW> pathParamValues;
+    public Map<String, String8FW> queryParamValues;
```

Comment:
The fields in this class are shared by all requests, except for `pathParamValues` and `queryParamValues` which are storing parsed values from an individual request, so should not be here.